v2.0/foreign-key.md

```diff
@@ -49,6 +49,18 @@ For example, if you have a Foreign Key constraint on columns `(A, B)` and try to
 
 However, allowing _NULL_ values in either your foreign key or referenced columns can degrade their referential integrity. To avoid this, you can use the [Not Null constraint](not-null.html) on both sets of columns when [creating your tables](create-table.html). (The Not Null constraint cannot be added to existing tables.)
 
+### Foreign Key Actions
+<span class="version-tag">New in v2.0:</span> You can specify a foreign key reference action.
+
+Parameter | Description
+----------|------------
+`ON DELETE NO ACTION` / `ON UPDATE NO ACTION` | _Default actions._ Fail if there are any existing references to the key being updated or deleted, but allow this check to be deferred until the end of the statement. <br><br>Aliases: `ON DELETE RESTRICT` / `ON UPDATE RESTRICT`
```

Comment:
In this table, I am unsure which actions I (the user) need to take and which actions CockroachDB performs that I am being informed about. For instance, "allow this check to be deferred" seems like something CockroachDB does (although I might be wrong), "the foreign key must be dropped and recreated" seems like something I would need to do (again, I might be wrong). Can we specify "you need to" or "CockroachDB does..." wherever possible?

Reply:
Edited language to be clearer.